<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shuriken Beat Slicer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_Play">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/media-playback-start.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/media-playback-start.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Stop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/media-playback-stop.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/media-playback-stop.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Original BPM:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_OriginalBPM">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_CalcBPM">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Calc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>New BPM:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_JackSync">
        <property name="text">
         <string>JACK Sync</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_NewBPM">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Apply">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_TimeStretch">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Time Stretch</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_PitchCorrection">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Pitch Correction</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_TimestretchOptions">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stretch Options...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Slice">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Slice</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/transform-move.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/transform-move.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Detection Method:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_DetectMethod">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Window Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_WindowSize">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Hop Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_HopSize">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Threshold:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_Threshold">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
            <property name="numDigits">
             <number>5</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
            <property name="value" stdset="0">
             <double>0.300000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_Threshold">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>300</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_FindOnsets">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Find Onsets</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_FindBeats">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Find Beats</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="WaveGraphicsView" name="waveGraphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Audio_File"/>
    <addaction name="actionExport_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAudio_Setup"/>
    <addaction name="actionUser_Interface"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionSelect_None"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Slice_Point"/>
    <addaction name="separator"/>
    <addaction name="actionApply_Gain"/>
    <addaction name="actionApply_Ramp"/>
    <addaction name="actionEnvelope"/>
    <addaction name="actionJoin"/>
    <addaction name="actionSplit"/>
    <addaction name="actionNormalise"/>
    <addaction name="actionReverse"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave_Project"/>
   <addaction name="actionClose_Project"/>
   <addaction name="separator"/>
   <addaction name="actionImport_Audio_File"/>
   <addaction name="actionExport_As"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionMove"/>
   <addaction name="actionSelect"/>
   <addaction name="actionAudition"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_Original"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionZoom_In"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_Audio_File">
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-import.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-import.png</iconset>
   </property>
   <property name="text">
    <string>Import Audio File</string>
   </property>
   <property name="toolTip">
    <string>Import Audio File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAudio_Setup">
   <property name="text">
    <string>Audio Setup</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-open.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="toolTip">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-save.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-close.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-close.png</iconset>
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionExport_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-export.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/document-export.png</iconset>
   </property>
   <property name="text">
    <string>Export As...</string>
   </property>
   <property name="toolTip">
    <string>Export As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/edit-undo.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/edit-redo.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionAdd_Slice_Point">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Slice Point</string>
   </property>
   <property name="toolTip">
    <string>Add Slice Point</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/edit-delete.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionReverse">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reverse</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionEnvelope">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Envelope</string>
   </property>
  </action>
  <action name="actionUser_Interface">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>User Interface</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/help-contents.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/help-about.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionJoin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Join</string>
   </property>
   <property name="shortcut">
    <string>J</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSelect_None">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select None</string>
   </property>
   <property name="toolTip">
    <string>Select None</string>
   </property>
  </action>
  <action name="actionApply_Gain">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Apply Gain</string>
   </property>
  </action>
  <action name="actionNormalise">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normalise</string>
   </property>
  </action>
  <action name="actionApply_Ramp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Apply Ramp</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/zoom-in.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/zoom-out.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_Original">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/zoom-original.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/zoom-original.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Original</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/edit-select.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/edit-select.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/kdenlive-select-all.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/kdenlive-select-all.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionSplit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Split</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionAudition">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../usr/share/icons/oxygen/22x22/actions/text-speak.png</normaloff>../../../../usr/share/icons/oxygen/22x22/actions/text-speak.png</iconset>
   </property>
   <property name="text">
    <string>Audition</string>
   </property>
   <property name="toolTip">
    <string>Audition</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WaveGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>src/wavegraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
